ebpf/c/tracer-ebpf.h

```diff
@@ -47,15 +55,18 @@ struct ipv6_tuple_t {
 	__u16 dport;
 	__u32 netns;
 	__u32 pid;
+	// Metadata description:
+	// First bit indicates if the connection is TCP (1) or UDP (0)
+	__u32 metadata; // This is that big because it seems that we atleast need a 32-bit aligned struct
 };
 
-#define TRACER_STATE_UNINITIALIZED 0
-#define TRACER_STATE_CHECKING      1
-#define TRACER_STATE_CHECKED       2
-#define TRACER_STATE_READY         3
+static const __u8 TRACER_STATE_UNINITIALIZED = 0;
```

Comment:
why these changes ?

Reply:
This ensures the value of theses constants can't be changed by error in the code